.devcontainer/contributor/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Radius CLI - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "1.21"
+        },
+        "ghcr.io/guiyomh/features/golangci-lint:0": {},
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+            "minikube": "none"
+        },
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {
+            "version": "latest",
+            "moby": true
+        }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "redhat.vscode-yaml",
+                "streetsidesoftware.code-spell-checker",
```

Comment:
What options do users have to install their own extensions when using a codespace/devcontainer? I'm asking because this isn't something we've adopted/documented as part of the project. Likewise `eamodio.gitlens`.

Reply:
I can add that one. In general, after the devcontainer/codespace started you can reinstall the additional extensions. However, as soon as you do a rebuild they are gone, so you must reinstall it

Reply:
They can install their own extension inside the container. However after a rebuild they are gone. 
I reduced the list to the minimum. Let me know if I should add/remove other extensions